scripts/upstream.sh

```diff
@@ -168,15 +168,15 @@ end_rebase() {
   rebase_merge_dir=$(cd upstream && git rev-parse --git-path rebase-merge)
   rebase_apply_dir=$(cd upstream && git rev-parse --git-path rebase-apply)
 
-  if [[ -d "${rebase_merge_dir}" || -d "${rebase_apply_dir}" ]]; then
+  if [ -d "${rebase_merge_dir}" ] || [ -d "${rebase_apply_dir}" ]; then
     echo "rebase still in progress in './upstream'. Please resolve the rebase in"
     echo "'./upstream' and then run 'make \"$1\"' again."
     exit 1
   fi
 
   rm patches/*.patch
   cd upstream
-  git format-patch local -o ../patches --no-prefix --zero-commit --no-signature --no-stat
+  git format-patch local -o ../patches --zero-commit --no-signature --no-stat
```

Comment:
is there a reason to remove --no-prefix?

Reply:
Yes. https://github.com/pulumi/ci-mgmt/pull/698/commits/88e3be43b4ec35e537a748333f301d31b4520950 has details.